scripts/setup

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh -e
```

Comment:
What's the purpose of this script?

Reply:
I'll give it a read when it's up and will let you know.

Reply:
You can read it [here](https://github.com/clairBuoyant/server/pull/40/files#diff-25b96aa16f8b9f68c692d0db061713707e9cfc67b296e0ea30908928fb3ffcd6R1-R29) before it's merged in order to incorporate any suggestions that you may have beforehand.